sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosItemRequestOptions.java

```diff
@@ -540,6 +539,9 @@ public void setThroughputControlGroupName(String throughputControlGroupName) {
      * @return  thresholdForDiagnosticsOnTracerInMS the latency threshold for diagnostics on tracer.
      */
     public Duration getThresholdForDiagnosticsOnTracer() {
+        if (this.thresholds == null) {
+            return Duration.ofMillis(100);
```

Comment:
not related to this PR -> but feel a little bit weird to return 100ms when no threshold is being defined on the itemRequestOptions, would it cause the same issue that client config being ignored? (also did not find this API being used anywhere)

Reply:
It is not used anywhere (and could be deprecated) - but it is public API. So, need to avoid NPE - and following what the comment is saying (again not really relevant because this API is not really used anymore (was meant to be used for old OTel trace code))